### PR TITLE
Add ImageDiff - Comparator of Two Images

### PR DESCRIPTION
**Pull request by garfieldduck**
- Add slidable image comparator using html input
- update webpack-isomorphic-tool to avoid problems related to sass files

### CODE REVIEW

File: src/components/article/ImageDiff.js [hunk 1]
```diff
@@ -38,22 +49,40 @@ class ImageDiff extends FitwidthMixin(Component) {
...
-      <div ref="imgDiff">
+      <div ref="imgDiff" className={styles.diffContainer} style={outerStyle}>
         <figure className={styles.wrapper}>
           <div className={styles.imgContainer}>
-            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-before.jpg" style={imgStyle}/>
+            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-after.jpg" style={imgStyle}/>
           </div>
-          <div>
-            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-before.jpg"/>
+          <div className={styles.overlayContainer} style={{ width: percentage+'%' }}>
```
Comment (hcchien) [inline src/components/article/ImageDiff.js]: is it still for developing?

Comment (hcchien) [inline src/components/article/ImageDiff.js]: is it still for developing?

Comment (garfieldduck) [inline src/components/article/ImageDiff.js]: yes. I will replace the url as the new version of imagediff of Keystone is being deployed.

File: src/lib/FitwidthMixin.js [hunk 2]
```diff
@@ -16,6 +17,15 @@ let FitwidthMixin = (superclass) => class extends superclass {
...
     }
   }
+
+  _getHeight(width, original, defaultWidth, defaultHeight) {
+    if (original) {
+      const oriWidth = _.get(original, 'width', defaultWidth)
+      const oriHeight = _.get(original, 'height', defaultHeight)
+      return Math.round(width * oriHeight / oriWidth)
+    }
+    return width
```
Comment (nickhsine) [inline src/lib/FitwidthMixin.js]: ```return defaultHeight``` ?

File: src/components/article/ImageDiff.js [hunk 3]
```diff
@@ -14,12 +15,22 @@ class ImageDiff extends FitwidthMixin(Component) {
...
-      width: 200
+      isMounted: false,
+      onhovered: false,
+      screenType: 'MOBILE',
+      width: 200,
+      placeholderOpacity: 0,
+      imageOpacity: 0,
+      percentage: 50
     }
     this.fitToParentWidth = this.fitToParentWidth.bind(this)
```
Comment (nickhsine) [inline src/components/article/ImageDiff.js]: do we need this binding?

File: src/components/article/ImageDiff.js [hunk 4]
```diff
@@ -38,22 +49,40 @@ class ImageDiff extends FitwidthMixin(Component) {
```
Comment (nickhsine) [inline src/components/article/ImageDiff.js]: do we need ```_renderFigure``` ?

File: src/components/article/ImageDiff.js (hunk 3 above)
Comment (garfieldduck) [inline src/components/article/ImageDiff.js]: yes

**Comment by nickhsine** [comment]
很讚！😎

**Pull request merged**